app/src/main/java/com/kinkong/QuestionActivity.java

```diff
@@ -28,22 +28,29 @@
 
     private static final float UNSELECTED_ALPHA = 0.7f;
     private static final float SELECTED_ALPHA = 1f;
+    private static final long WAIT_TIME_BETWEEN_QUESTIONS = 10000;
 
     private Question question;
     private KinAccount account;
     private MediaPlayer mediaPlayer;
-
     private int answerIndex = UNSELECTED_INDEX;
+    private View root;
 
     private List<AnswerView> answerViewList = new ArrayList<>(4);
+    private boolean isWinner;
+    private boolean shouldMoveToScreen;
+    private boolean hasNextQuestion;
+
+
     private View.OnClickListener answerClickListener = view -> {
         AnswerView answerView = (AnswerView) view;
         answerView.setSelected(true);
         disableClicks();
         answerIndex = Integer.parseInt((String) view.getTag());
         updateSelection(answerIndex);
+        isWinner = answerIndex == question.getCorrectAnswer();
```

Comment:
parenthesis here can be helpful

Reply:
answerIndex no longer needs to be a member variable. it can be a local variable and be passed to sendAnswer as a parameter